src/backend/distributed/planner/local_distributed_join_planner.c

```diff
@@ -530,7 +557,13 @@ RequiredAttrNumbersForRelationInternal(Query *queryToProcess, int rteIndex)
  */
 static ConversionCandidates *
 CreateConversionCandidates(PlannerRestrictionContext *plannerRestrictionContext,
```

Comment:
let's try to avoid ifdefs in the function signatures, they seem super hard to follow